src/chrome/chromeDebugAdapter.ts

```diff
@@ -1203,19 +1203,23 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
             return;
         }
 
-        const committedBps = this._committedBreakpointsByUrl.get(script.url) || [];
+        // committed breakpoints (this._committedBreakpointsByUrl) should always have url keys in canonicalized form
+        const committedBps = this.getValueFromCommittedBreakpointsByUrl(script.url);
```

Comment:
What will happen if there is no commitedBps for this url?

Reply:
Sorry I forgot to put back the empty array